### PR TITLE
Food template

### PR DESCRIPTION
**Pull request by SAHU-01**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

A new food template was added to the website.

Fixes # (issue no.)

#144 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Explain the Testing instructions

**Test Configuration**:

- Operating system: Windos
- Version: 10
- Text-editors used: VSCode

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
image
image
image
image

### CODE REVIEW

**Review by arpit456jain — CHANGES_REQUESTED** [review]
I had requested some changes kindly have a look on them

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5503
+}
```
Comment (arpit456jain) [inline .vscode/settings.json:3]: remove this .vs code file 
if its not in git ignore file please add it

**Review by arpit456jain — CHANGES_REQUESTED** [review]

File: style_links.json [hunk 2]
```diff
@@ -124,6 +124,11 @@
     "name": "Yoga Template 2",
     "link": "Yoga Template 2/index.html"
   },
+
+  {
+    "name": "Food Template New",
+    "link": "Food_Template/index.html"
+  },
```
Comment (arpit456jain) [inline style_links.json:132]: add it on appropriate place (alphabetically)

**Review by arpit456jain — CHANGES_REQUESTED** [review]

File: style_links.json (hunk 2 above)
Comment (SAHU-01) [reply style_links.json:132]: sure, i'll have to remove the vscode file and add it alphabetically  ,anything else?

Comment (arpit456jain) [reply style_links.json:132]: nope

Comment (SAHU-01) [reply style_links.json]: done ,know you can give it a look!

**Review by arpit456jain — APPROVED** [review]

**Comment by arpit456jain** [comment]
great work @SAHU-01 its amazing 🎉🎉

**Pull request merged**